Source/DlgSystem/GameplayDebugger/SDlgDataDisplay.cpp

```diff
@@ -31,7 +31,7 @@ void SDlgDataDisplay::Construct(const FArguments& InArgs, const TWeakObjectPtr<c
 	WorldContextObjectPtr = InWorldContextObjectPtr;
 	RootTreeItem = MakeShared<FDlgDataDisplayTreeRootNode>();
 	ActorsTreeView = SNew(STreeView<TSharedPtr<FDlgDataDisplayTreeNode>>)
-		.ItemHeight(32)
+		//.ItemHeight(32)
```

Comment:
Maybe add comment why this is commented or just remove?

Reply:
removed the code